app/controllers/feedback_controller.rb

```diff
@@ -0,0 +1,31 @@
+class FeedbackController < ApplicationController
+  def new
+    if Rails.env.development? || Rails.env.test?
```

Comment:
this is temporary code — we should add a TODO when we add code that's obviously meant to be removed soon, so we don't forget